### ISSUE TITLE
Add config for JSON file log directory (customize cypress/logs/)

### ISSUE THREAD

**Issue by MadLittleMods** [bahmutov/cypress-failed-log#159]
Add config for where the [JSON file log](https://github.com/bahmutov/cypress-failed-log#json-file) is saved. Currently it is hard-coded for `cypress/logs/`

Similar to all of the config with Cypress `screenshotsFolder`, `videosFolder`, https://docs.cypress.io/guides/references/configuration.html#Folders-Files

**Comment by MohamadKh75** [comment]
any updates @bahmutov?

**Comment by bahmutov** [comment]
Nope, I don't see anyone opening a pull request to add this feature
